.github/workflows/validate-pr-release-info.yml

```diff
@@ -16,6 +16,7 @@ on:
 jobs:
   check-pr:
     name: Validate Release Label and Notes
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'release') || !(contains(github.event.pull_request.labels.*.name, 'tooling') || contains(github.event.pull_request.labels.*.name, 'documentation') || contains(github.event.pull_request.labels.*.name, 'restructure'))}}
```

Comment:
Just to clarify, do we want to run the workflow when no labels have been added? Or do we only want to run when a release label is present?

Reply:
My intention is for it to run when no labels have been added so that the check will fail, encouraging us to always add labels to PRs

Reply:
I pulled out a couple environment variables to make the condition more readable